service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProviderTest.java

```diff
@@ -0,0 +1,166 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+import uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences;
+import uk.gov.hmcts.reform.fpl.model.CaseManagementOrder;
+import uk.gov.hmcts.reform.fpl.model.Judge;
+import uk.gov.hmcts.reform.fpl.model.OrderAction;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.utils.AssertionHelper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.ISO_8859_1;
+import static org.apache.commons.lang3.RandomUtils.nextBytes;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.CMOStatus.SEND_TO_JUDGE;
+import static uk.gov.hmcts.reform.fpl.service.email.content.AbstractEmailContentProviderTest.BASE_URL;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class, CaseManagementOrderEmailContentProvider.class})
+@TestPropertySource(properties = {"ccd.ui.base.url=" + BASE_URL})
+class CaseManagementOrderEmailContentProviderTest extends AbstractEmailContentProviderTest {
+
+    private static final Long CASE_ID = 12345L;
+
+    @Autowired
+    private CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+
+    @Test
+    void shouldBuildCMOIssuedCaseLinkNotificationExpectedParameters() {
+        Map<String, Object> expectedParameters = ImmutableMap.<String, Object>builder()
+            .put("localAuthorityNameOrRepresentativeFullName", "testName")
+            .put("caseUrl", buildCaseUrl())
+            .put("subjectLineWithHearingDate", "lastName, 11")
+            .put("reference", "12345")
+            .build();
+
+        assertThat(caseManagementOrderEmailContentProvider
+            .buildCMOIssuedCaseLinkNotificationParameters(createCase(), "testName"))
+            .isEqualTo(expectedParameters);
+    }
+
+    @Test
+    void shouldBuildCMOIssuedWithoutDocumentLinkNotificationExpectedParameters() {
+
+        Map<String, Object> expectedParameters = ImmutableMap.<String, Object>builder()
+            .put("cafcassOrRespondentName", "testName")
+            .put("caseUrl", buildCaseUrl())
+            .put("subjectLineWithHearingDate", "lastName, 11")
+            .put("reference", "12345")
```

Comment:
I think we might want to have these as constants as they are now in two different methods?